### PR TITLE
Plugin Maintenance update

### PR DESCRIPTION
**Pull request by danishxtendify**
**Clickup Task:** https://app.clickup.com/t/8jqacn 

**Description:** 
* Fix: Minify JS.
* Update: Compatibility check with latest WordPress v5.5.1.
* Update: Compatibility check with Latest WP Job Manager v1.34.3.
* Update: Compatibility check with the latest PHP v7.4.10.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -2,6 +2,15 @@
 module.exports = function(grunt) {
 
 	grunt.initConfig({
+		pkg : grunt.file.readJSON( 'package.json' ),
+		
+		uglify : {
+			task1: {
```
Comment (rosinghal) [inline Gruntfile.js:8]: @danishxtendify `task1` is not correct.

File: readme.txt [hunk 2]
```diff
@@ -31,6 +31,13 @@ Astoundify has released two themes that are fully integrated with the WP Job Man
 
 == Changelog ==
 
+= 1.17.5: Oct 27, 2020 =
+
+* Fix: Minify js.
```
Comment (rosinghal) [inline readme.txt:36]:
```suggestion
* Fix: Minify JS file.
```

File: Gruntfile.js (hunk 1 above)
Comment (danishxtendify) [reply Gruntfile.js]: @rosinghal repleaced with assets_js

File: readme.txt [hunk 3]
```diff
@@ -31,6 +31,13 @@ Astoundify has released two themes that are fully integrated with the WP Job Man
 
 == Changelog ==
 
+= 1.17.5: Oct 27, 2020 =
+
+* Fix: Minify JS file.
+* Update: Compatibility check with latest WordPress v5.5.1.
+* Update: Compatibility check with Latest WP Job Manager v1.34.3.
+* Update: Compatibility check with the latest PHP v7.4.10.
```
Comment (rosinghal) [inline readme.txt:39]: Please mention in changelog about https://github.com/Astoundify/wp-job-manager-regions/pull/66.

Comment (danishxtendify) [reply readme.txt:40]: @rosinghal update a few days ago.

**Review by rosinghal — APPROVED** [review]

File: package.json [hunk 4]
```diff
@@ -1,13 +1,15 @@
 {
   "name": "wp-job-manager-regions",
-  "version": "1.17.4",
+  "version": "1.17.5",
```
Comment (rosinghal) [inline package.json:3]:
```suggestion
  "version": "1.17.7",
```

File: readme.txt [hunk 5]
```diff
@@ -5,8 +5,8 @@ Donate link: https://www.paypal.me/astoundify
 Contributors: Astoundify
 Tags: job, job listing, job region
 Requires at least: 4.7.0
-Tested up to: 5.0.3
-Stable Tag: 1.17.4
+Tested up to: 5.5.1
+Stable Tag: 1.17.5
```
Comment (rosinghal) [inline readme.txt:9]:
```suggestion
Stable Tag: 1.17.7
```

File: readme.txt [hunk 6]
```diff
@@ -31,6 +31,14 @@ Astoundify has released two themes that are fully integrated with the WP Job Man
 
 == Changelog ==
 
+= 1.17.5: Oct 27, 2020 =
```
Comment (rosinghal) [inline readme.txt:34]:
```suggestion
= 1.17.7: Oct 27, 2020 =
```

File: wp-job-manager-locations.php [hunk 7]
```diff
@@ -5,7 +5,7 @@
  * Description: Create predefined regions/locations that job submissions can associate themselves with.
  * Author:      Astoundify
  * Author URI:  http://astoundify.com
- * Version:     1.17.4
+ * Version:     1.17.5
```
Comment (rosinghal) [inline wp-job-manager-locations.php:8]:
```suggestion
 * Version:     1.17.7
```

**Pull request merged**